src/main/MQ2ImGuiConsole.cpp

```diff
@@ -889,6 +891,18 @@ struct ImGuiZepConsole : public mq::imgui::ConsoleWidget, public mq::imgui::ImGu
 		ImGuiZepEditor::Notify(message);
 	}
 
+	int GetConsoleFontSize() const override
+	{
+		return m_fontSize;
+	}
+
+	void SetConsoleFontSize(int newFontSize) override
+	{
+		m_fontSize = newFontSize;
+		SetFont(Zep::ZepTextType::Text, mq::imgui::ConsoleFont, m_fontSize);
+	}
```

Comment:
The console prefix in these names is redundant.

this still needs tonly be called if the font size actually changes.